src/core/desktoputils.hpp

```diff
@@ -0,0 +1,7 @@
+#pragma once
+
+#include <qstringlist.h>
+
+namespace DesktopUtils {
+QList<QString> getDesktopDirectories();
```

Comment:
I am not sure why this was extracted into a new file

Reply:
Moved to `desktopentrymonitor.cpp`